baseline.gradle

```diff
@@ -48,6 +49,8 @@ subprojects {
   // import order.
   pluginManager.withPlugin('com.diffplug.spotless') {
     spotless {
+      // don't run spotlessCheck during gradle check task
+      enforceCheck = false
```

Comment:
So is this just something we run ourselves or when does it get run?

Reply:
so previously the gradle spotless plugin was already there, but never enabled. This PR enables it and makes sure it doesn't run during the normal `check` of Gradle because it will complain that the code format is off. 
The implication of this is that we would run `spotlessCheck` / `spotlessApply` manually for now until the format of the codebase is in a state where we can enforce this on CI. Bringing the codebase into this state is what I wanted to focus on in a separate PR/discussion and I started already talking to @rdblue about it.

Reply:
That makes sense to me.

Thank you for filling me in on the details as I've had less bandwidth for doing reviews than I normally do. Makes sense to add it in and then slowly updated over time. I'll be sure to remind people ocassionally to run `spotlessCheck` during PRs.

Reply:
Additional work needs to happen (which I'm looking into) before we can have `spotlessCheck` produce the actual code format that is desired btw :)